src/components/WorkElement/TradeDataList.js

```diff
@@ -10,6 +10,7 @@ const TradeDataList = ({ trades, register, errors, onTradeSelect }) => {
     <DataList
       name="trade"
       label="Trade"
+      labelMessage="- Search by type (e.g. Gas) or code (e.g.GS)"
```

Comment:
Tiny nitpick:
I know the screenshot has it like you have it here but should there not be a space between `e.g.` and `GS` like we have in `e.g. Gas` ?
Seems inconsistent otherwise